### DISCUSSION TITLE
Larger or full-screen post editor

### DISCUSSION THREAD

**Discussion by chenilim** [mattermost/community#67] [General Ideas]
I'm sure this must have come up before, but I often want a larger input box to compose posts. I typically use an external editor for that and paste in, especially when composing status reports or posts that use a lot of Markdown.

At a minimum, it would be great to have a "pop-up" button that shows the edit box as a form dialog. Bonus points if that had a markdown toolbar.

**Comment by esethna** [comment]
+1, @matthewbirtch was this considered at all in the advanced text editor design, I can't recall?

**Reply by matthewbirtch** [reply]
Actually, no. It's a great idea though!
